### DISCUSSION TITLE
Ability to Change Email Address for Cloud/Backoffice Users

### DISCUSSION THREAD

**Discussion by nobledm** [umbraco/Umbraco-CMS#14548] [Features and ideas]
Currently it is not possible to change a user's email address in a Cloud project. This may not come up much, but when it does it can be frustrating for clients to have to use a brand new account to continue using their site. In many cases it also leaves inactive accounts hanging around to avoid broken links for things like news/blog authors for example.

It could be a nice feature to allow email changes one day :3

**Comment by sajumb** [comment]
Thank you for bringing up your concern regarding the inability to change email addresses in the Umbraco Cloud backoffice. We understand that this limitation can be inconvenient for users, and I would like to assure you that we are actively working on a solution.

We have recognized the importance of allowing users to change their email addresses and have initiated a project to address this issue. Our development team is working diligently to enable this feature in Umbraco Cloud, and we are confident that it will have a positive impact not only on Umbraco Cloud but also on the Umbraco backoffice when running in the cloud environment.

While we cannot provide an exact timeline for the update at this moment, we expect to have a resolution ready for you by next year.

kind regards
Søren, Product owner of Umbraco Cloud

**Comment by bjarnef** [comment]
I have had this need as well on a few Umbraco Cloud projects in the past. It would also ensure the history is maintained for the specific user and we avoid to have redundant users accounts for same content editor.

From time to time companies are rebranded, which also mean the domain and email addresses may be updated.
